src/morello-issues/README.md

```diff
@@ -3,10 +3,42 @@
 The following known issues may affect the reliability and usability of
 CheriBSD on Morello:
 
+- If a USB hub (including one in a USB keyboard) the system will pause on boot
+  due to a firmware bug.  The pause can be bypassed by pressing Enter on a USB
+  keyboard attached to the Morello desktop system.  Once Enter is pressed the
```

Comment:
So it varies by device.  George can press Enter on his keyboard.  For Robert and I this doesn't work and we have to unplug the keyboard entirely.